components/pedigree/resources/src/main/resources/pedigree/view/personVisuals.js

```diff
@@ -1055,6 +1055,12 @@ define([
                 }
             }
 
+            // move white background up if it is not the first child element of <svg>, otherwise it will hide labels
```

Comment:
This seems misplaced, why does it have to be done in the per-node `drawLabels`? The background is global, so it should, in theory, be placed as the first element only once. If it doesn't stay there, I'd say that other code is buggy, and this is just a hacky fix.